README.md

```diff
@@ -1,29 +1,170 @@
-# Waku - Idea Board
-
-
-- Polling/Voting: Enable polling or voting off-chain, vote results could be aggregated and submitted on chain in one transaction.
-- Marketplace (buy/sell goods or NFTs, ride-share, etc): Send offer and negotiate off-chain. Highest bids sent over Waku could be binding and submitted to the contract by seller.
-- Collaborative editing: Use Waku for live collaborative editing of documents, saved version of documents can then be stored on decentralized storage.
-- Multiplayer games: Waku can be used to communicate game move off-chain, final state (e.g. winner) can then be submitted on chain for reward (e.g. NFT mint or winner takes stake).
-- IoT systems: Enable devices to communicate or report small data payload in a decentralized manner.
-- Decentralized wallet address ownership verification: Use Waku to enable communication between dApp and wallet such as signature or zeroknowledge proof exchange to prove identity.
-- Notifications centre: use an SDK (probably go-waku?) to build general Notification protocol over Waku and a mobile app allowing you to replace centralized Push Notifications provided by Apple/Google
-- API generator: build a tool to generate a Waku protocol (and code) from a provided OpenAPI specification 
-- Leader Election Protocol: Leader election can viewed as a soft consensus mechanism. Implement something like [RAFT](https://raft.github.io/) as a library on top of Waku (using any SDK), so that users can easily plug it in their applications
-- Reputation systems:
-- Censorship resistant reviews plugin:
-- Privacy preserving location tracker:
-- Medical systems for remote diagnosis :
-- Decentralized smart grid system :
-- Decentralized autonomous vehicle coordination :
-- Crowd-sourced weather data network :
-- P2P Chess :
-- P2P TicTacToe :
-- Privacy focused fitness tracker :
-- Decentralized and privacy-focused hiring platform (Web3 version of BambooHR) :
-- DAO/Governance Tooling :
-- Satellite communication systems :
-- Decentralized web Walkie-Talkie :
-- Crypto ATMs :
-- Smart access cards :
-- Chat app: Proper chat protocol is complex, we recommend to instead wait for our upcoming Chat SDK.
+# Waku - Idea Board 💡
+
+Below is a compilation of innovative project ideas curated by the Waku community. These ideas cover a wide range of use cases and applications, showcasing the versatility and potential of the Waku protocol for decentralized, privacy preserving communication and beyond.
+
+<br>
+
+> Note : Numerous traditional chat applications have already harnessed Waku's capabilities. To truly innovate and avoid re-inventing the wheel, we encourage hackers to explore untapped possibilities and unique projects that integrate Waku in other novel ways apart from chat.
+
+<br>
+
+<details>
+<summary><b>Polling/Voting</b></summary>
+<br>
+Enable polling or voting off-chain, vote results could be aggregated and submitted on-chain in one transaction.
```

Comment:
Or not even need to submit on chain, and just do it all off-chain, users can aggregate poll responses from Waku to see votes.